_assets/css/_sidebar.scss

```diff
@@ -33,82 +29,106 @@ Stylesheet: Side Bar Page Stylesheet
   }
 }
 
+.site.sidebar.scroll-menu-enabled .site-header {
+  position: fixed;
+}
+
 // Side Nav
 
 $sidebar-nav-width: rem-calc(240);
 
-.site.sidebar .sidenav {
-  ul {
-    list-style: none;
-    margin: 0;
-  }
+.site.sidebar {
+  background-color: $lighter-gray;
 
-  li {
-    margin: 0;
-  }
+  @include breakpoint(medium down) {
+    &:not(.scroll-menu-enabled) {
+      .site-wrapper {
+        padding-top: 0;
+      }
 
-  a {
-    color: inherit;
+      .sidenav {
+        position: sticky;
+        top: 0;
+      }
+    }
 
-    &:hover {
-      color: $primary-color;
+    &.scroll-menu-enabled .sidenav {
+      height: rem-calc(90);
+      left: 0;
+      padding-top: rem-calc(40);
+      position: fixed;
+      top: rem-calc(-40);
+      transform: translateY(#{rem-calc(40)});
+      width: 100%;
     }
   }
 
-  .is-active,
-  .has-active-children {
-    > a:hover {
-      text-decoration: none;
+  .sidenav {
+    ul {
+      list-style: none;
+      margin: 0;
     }
-  }
 
-  .primary > li:not(.is-active):not(.has-active-children) {
-    display: none;
-  }
+    li {
+      margin: 0;
+    }
 
-  .section {
-    display: none;
-  }
+    a {
+      color: inherit;
 
-  @include breakpoint(large) {
-    position: fixed;
-    top: rem-calc(64);
-    left: 0;
-    flex: 0 0 #{$sidebar-nav-width};
-    width: $sidebar-nav-width;
-    height: calc(100% - #{rem-calc(64)});
-    border-right: 1px solid $medium-light-gray;
-    overflow-y: auto;
-    overflow-x: hidden;
-    transition: top 0.4s ease-in-out, height 0.4s ease-in-out;
-  }
+      &:hover {
+        color: $primary-color;
+      }
+    }
 
-  @include breakpoint(medium down) {
-    position: fixed;
-    z-index: 19;
-    left: 0;
-    top: rem-calc(-40);
-    padding-top: rem-calc(40);
-    transform: translateY(#{rem-calc(40)});
-    width: 100%;
-    height: rem-calc(90);
-    transition: transform 0.4s ease-in-out, height 0.4s ease-in-out;
-    overflow-y: auto;
-    background-color: $lighter-gray;
-    box-shadow: $element-shadow;
-
-    .primary > .is-active.has-children,
-    .primary > .has-active-children {
-      .subfolder {
-        > a {
-          display: none;
-        }
+    .is-active,
+    .has-active-children {
+      > a:hover {
+        text-decoration: none;
+      }
+    }
+
+    .primary > li:not(.is-active):not(.has-active-children) {
```

Comment:
![](https://media2.giphy.com/media/tzv60mgyXW3GE/giphy.gif)

Reply:
_Note: a lot of this is just moved._ 

/me hides.